sdk/src/main/scala/com/horizen/account/mempool/TxCache.scala

```diff
@@ -69,14 +94,49 @@ class TxCache {
 
   def getSizeInSlots: Int = sizeInSlots
 
+  def getNonExecSizeInSlots: Int = nonExecSizeInSlots
+
   def getOldestTransaction(): Option[SidechainTypes#SCAT] = oldestTx.map(_.tx)
 
   def getYoungestTransaction(): Option[SidechainTypes#SCAT] = youngestTx.map(_.tx)
 
+  def getNonExecIterator(): NonExecTransactionIterator = new NonExecTransactionIterator
```

Comment:
let's put a comment to the method: from oldest to youngest

Reply:
Done